Sitenotice.php

```diff
@@ -10,19 +10,19 @@
 // and don't comment it out
 $wgMajorSiteNoticeID = 56;
```

Comment:
The comment above suggests this ID must be changed

Reply:
Thanks, done.